manoseimas/mps_v2/models.py

```diff
@@ -74,16 +75,16 @@ def __unicode__(self):
 
 class Group(CrawledItem):
     GROUP_TYPES = (
-        ('group', ('Group')),
-        ('committee', ('Committee')),
-        ('commission', ('Commission')),
-        ('fraction', ('Fraction')),
-        ('parliament', ('Parliament')),
+        ('group', _('Group')),
+        ('committee', _('Committee')),
+        ('commission', _('Commission')),
+        ('fraction', _('Fraction')),
+        ('parliament', _('Parliament')),
```

Comment:
All translation strings in global scope should use `ugettext_lazy` [1], otherwise users will not be able to switch translation, because all messages are translated when process is started (no per request translation).

[1] https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#lazy-translation